smoke-tests/apps/SamplingOverrides/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/ExceptionWithSamplingOverridesTest.java

```diff
@@ -53,9 +52,6 @@ static class Tomcat8Java17Test extends ExceptionWithSamplingOverridesTest {}
   @Environment(TOMCAT_8_JAVA_19)
   static class Tomcat8Java19Test extends ExceptionWithSamplingOverridesTest {}
 
-  @Environment(TOMCAT_8_JAVA_20)
-  static class Tomcat8Java20Test extends ExceptionWithSamplingOverridesTest {}
-
   @Environment(TOMCAT_8_JAVA_21)
   static class Tomcat8Java21Test extends ExceptionWithSamplingOverridesTest {}
```

Comment:
looks like Java 21 OpenJ9 could be added to this and the smoke tests below also

Reply:
i added to this class. didn't add it for the test.. java 20 didn't have it. it will increase the PR build time?

Reply:
this class already has `TOMCAT_8_JAVA_8_OPENJ9` and `TOMCAT_8_JAVA_11_OPENJ9`

Reply:
> it will increase the PR build time?

yes, it does

Reply:
we can remove all Java 19 tests since Java 19 isn't an LTS release